src/ds/json.h

```diff
@@ -61,19 +62,49 @@ namespace std
   template <typename T>
   inline void to_json(nlohmann::json& j, const std::vector<T>& t)
   {
-    j = nlohmann::json::array();
-    for (const auto& e : t)
+    if constexpr (std::is_same_v<T, uint8_t>)
```

Comment:
It looks like we don't have an equivalent path in the schema generation, so where we're doing this automatic conversion to a base64 string the schema will still claim a list of uint8s?

Reply:
Done now. The associated strings are marked as "base64" in the OpenAPI spec too.